### PR TITLE
Added 3 new events (page layout prototype)

### PR DESCRIPTION
**Pull request by hauten**
Added one past and two future events to the page with style for date/location. Note that ASC S3C is now postponed, so I added parenthetical status to the events. See what you think; happy to modify as desired. Screen shot below.
Events mockup

### CODE REVIEW

File: events.md [hunk 1]
```diff
@@ -4,7 +4,18 @@ author_profile: false
...
 ---
 
-Coming soon: this page will have a list of past and upcoming Spack talks,
-tutorials, webinars, and other events of interest.
+## 2020
 
-**For now, check out our page on [Spack at SC19](/spack-at-sc19/)**.
+### [ISC High Performance](https://www.isc-hpc.com/)
+<small>21-25 June 2020 | Frankfurt, Germany (PENDING)</small>  
+The ISC (formerly known as the International Supercomputing Conference) caters to the hardware and software demands of global research centers and businesses in the fields of HPC, artificial intelligence, machine learning and data analytics.
```
Comment (tgamblin) [inline events.md]: Let's not put this one in until we get confirmation of acceptance.

File: events.md [hunk 2]
```diff
@@ -4,7 +4,18 @@ author_profile: false
...
+
+### [ASC S3C](https://s3c.sandia.gov/)
+<small class="pull-right">14-16 April 2020 | Albuquerque, NM (POSTPONED)</small>  
+The Tri-lab Advanced Simulation & Computing Sustainable Scientific Software Conference connects individuals and teams from the ASC Tri-labs that are working to deliver scientific software solutions in a sustainable manner. Greg Becker will be presenting a session on Spack.
```
Comment (tgamblin) [inline events.md]: a 20-minute presentation and a half-day tutorial

**Review by tgamblin — CHANGES_REQUESTED** [review]

File: events.md (hunk 2 above)
Comment (hauten) [reply events.md]: Added in a21b820

File: events.md (hunk 1 above)
Comment (hauten) [reply events.md]: Sorry, somehow didn't see these notes until today. Fixed in a21b820

**Comment by hauten** [comment]
Forgot to remove SC19 from the main nav in commits before today

**Review by tgamblin — APPROVED** [review]

**Comment by tgamblin** [comment]
Thanks!

FYI: we did get accepted at ISC so we can put the BOF and tutorial up now

**Pull request merged**